.gitignore

```diff
@@ -52,3 +52,6 @@ modules/orm
 deployment.env
 docker.env
 vault.txt
+
+#Packages development
+packages
```

Comment:
What is the purpose of this `packages` folder?

Reply:
ohh ... I could remove this, I initially had a packages folder for package development in the platform root directory.